integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -1139,6 +1135,12 @@ object BloopDefaults {
     BloopKeys.bloopGenerate
       .all(filter)
       .map(_.toSet)
+      .map(_.map {
+        case Inc(th) =>
+          logger.error(s"Couldn't run bloopGenerate. Cause:\n$th")
```

Comment:
So this should still allow the behaviour introduced in https://github.com/scalacenter/bloop/pull/1663 ? Meaning, fail, but still run bloop generate for all the projects it could.

Reply:
Hmm, nope, that behaviour wont work after. Should we introduce some flag to bloopInstall to allow #1663 behaviour, cause it's look like failing import is preferred action?

Reply:
maybe we should introduce some new task which one should be run manually for #1663 behaviour

Reply:
Guess we can do some changes to keep both behaviours, I'll try

Reply:
Thanks!

Reply:
I'm not sure about how tasks evaluate, I've read some time ago that there's no order guarantees. If at this point of evaluation all bloopGenerate tasks finished then #1663 is satisfied

Reply:
Could we maybe just flatten the result first? Or do a fold on it? And once we get the final result we can throw the exception.

Reply:
Just jumping in here to saw that I agree with @tgodzik. I think it's importing to keep the behavior to complete as much as possible and to continue to the end. Once we have the collection of results of all the `blooGenerate`s we should then be able to still determine if there was a single `Inc` and if so report on it.

Reply:
I made some changes, need your approve on that

Reply:
Done! Sorry for the delay!